docs/pills/09-disposability.md

```diff
@@ -0,0 +1,39 @@
+# [Disposability](https://12factor.net/disposability)
+
+::: tip ***Maximize robustness with fast startup and graceful shutdown***
+An app should be stopped gracefully at any time without losing data or causing disruptions. It should also start up fast.
+:::
+<br>
+
+![Disposability](../assets/disposability.png){ style="display: block; margin: 0 auto" }
+
+In a resilience and robust system that scales dynamically, an app has to be replaceable fast and easily. I must therefore be able to handle a ``SIGTERM``, completing running processes and freeing the resources as quick as possible. In the same way, the start-up of the app should be equally fast and completed as soon as possible.
+
+## Reduce the potential downtime between updating or scaling events
+
+This means new instances should be started, traffic gradually shifted, and old instances gracefully terminated without interruptions. Too slow a response to a health check at end or start could lead to alerts and deployment failures in certain systems.
```

Comment:
I'm not really sure about understanding this

`Too slow a response to a health check at end or start ...`

Reply:
DeepL says it's correct, but I changed it if feels weird